articles/extensions/bitbucket-deploy.md

```diff
@@ -0,0 +1,128 @@
+
+## Deployment
+
+Once you have set up the webhook in Bitbucket using the provided information, you are ready to start committing to your repository.
+
+With each commit you push to your configured Bitbucket repository, the webhook will call the extension to initiate a deployment if changes were made to the `rules` and/or the `database-connection` folders.
+
+The **Deploy** button on the **Deployments** tab of the extension allows you to manually deploy the Rules and Database Connection scripts that you already have in your Bitbucket repository. This is useful if your repository already contains items that you want to deploy once you have set up the extension or if you have accidentally deleted some scripts in Auth0 and need to redeploy the latest version of your repository.
+
+::: panel-warning Full Deployment
+To maintain a consistent state, the extension will always do a full deployment of the contents of these folders. Any rules or database connection scripts that exist in Auth0 but not in your GitHub repository will be **deleted**.
+:::
+
+### Deploy Database Connection Scripts
+
+To deploy Database Connection scripts, you must first create a directory under `database-connections`. The name of the directory must match **exactly** the name of your [database connection](${uiURL}/#/connections/database) in Auth0. You can create as many directories as you have Database Connections.
+
+Under the created directory, create one file for each script you want to use. The allowed scripts are:
+
+- `get_user.js`;
+- `create.js`;
+- `verify.js`;
+- `login.js`;
+- `change_password.js`;
+- `delete.js`.
+
+For a generic Custom Database Connection, only the `login.js` script is required. If you enable the migration feature, you will also need to provide the `get_user.js` script.
+
+You can find examples in [the Auth0 Samples repository](https://github.com/auth0-samples/github-source-control-integration/tree/master/database-connections/my-custom-db). While the samples were authored for GitHub, it will work for a Bitbucket integration as well.
+
+### Deploy Rules
+
+To deploy a rule, you must first create a JavaScript file under the `rules` directory of your Bitbucket repository. Each Rule must be in its own `.js` file.
+
+For example, if you create the file `rules/set-country.js`, the extension will create a Rule in Auth0 with the name `set-country`.
+
+**NOTE**: If you plan to use source control integration for an existing account, first rename your Rules in Auth0 to match the name of the files you will be deploying to this directory.
+
+You can control the Rule order, status (`enabled`/`disabled`) and stage (`login_success`, `login_failure`, `user_registration`) by creating a JSON file with the same name as your JavaScript file. For this example, you would create a file named `rules/set-country.json`.
```

Comment:
For now we'll only cover `login_success`. The sample repository will also only cover login success for now.